tests/probes/test_probes_leakreplay.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import garak
+import garak._config
+import garak._plugins
+import garak.attempt
+import garak.cli
+
+
+def test_leakreplay_hitlog():
+
+    args = "-m test.Blank -p leakreplay -d always.Fail".split()
+    garak.cli.main(args)
+
+
+def test_leakreplay_output_count():
+    generations = 1
+    garak._config.load_base_config()
+    garak._config.transient.reportfile = open("/dev/null", "w+")
```

Comment:
`/dev/null` is not a thing on windows consider setting a temp file here.

Reply:
changed to `os.devnull`